### ISSUE TITLE
Create a wiki for this project

### ISSUE THREAD

**Issue by johnwinans** [johnwinans/2063-Z80-cpm#1]
Please submit a comment on this thread if you would like to participate in the creation of a wiki for this project.

The goal will be to put together a web site version of my youtube series where I discuss this project.

I have never created one in GitHub before.  It seems easy enough.  But it might help if someone that has done one before could share their insights on organizing it.

Thank you.

**Comment by joelburton** [comment]
I'd be very happy to help out here.

I don't have a background in EE (I teach software engineering for a living). While I've been dabbling in computer electronics for a while, I've found the videos in this project to be very helpful in learning how to think through a design like this. However, when it comes to building and debugging things, having some written notes to follow will be helpful, since it's tricky to remember which part of which video covers everything.

I'm happy to make a straw proposal for organizing and help out with the writing. How about:

```
Intro
  ProjectGoals
Components
Building
Debugging
Serial
Printers
Cpm
ProgrammingBoard
RelatedProjects
```

**Comment by johnwinans** [comment]
I agree.

This started as a trip down memory lane.  As I see notes from people that are building their own boards, I'd like to make sure that they have more than the videos to refer back to.  Because they are annoying to rummage through if you are looking for something specific and can't recall where it was.

I figured I'd take a pass at skimming through the videos and creating an as-built outline of what is there so that a list like yours can connect to pages with a descriptions, pictures, and links to the videos if anyone wants more.

Do you have experience with Wikis and the like?  I wouldn't know if a github wiki is optimal for this or not.

Any thoughts?

**Comment by Stefanie80** [comment]
I'd like to bump this idea again. 
Would it be possible to use the Wiki functionality here on github?
There has been so much information shared in the discord lately, 
But that will be hard to find in a week, let alone a year.

**Comment by johnwinans** [comment]
You are right.  This has fallen off the radar.  The discord is not viable as a place to collect information.

I have never done a wiki on github.  Would you be interested working on a first draft?

**Comment by mchobby** [comment]
Is this point still open?
I did follow the project since 1Year+ and learn enough to start Z80 programming on a a salvaged Z80 board.

I do use a  MediaWiki for my company to share openSource content, translation to French and electronic learning stuff.
I started using the Wiki 10 years before and create some useful trick like:
- welcome page with dedicated section (MicroPython, Arduino, etc, etc) --> [https://wiki.mchobby.be](https://wiki.mchobby.be)
- section page with "boxed" entry point to the various tutorial. It is acting like "chapters" in a book --> [https://wiki.mchobby.be/index.php?title=MicroPython-Accueil](https://wiki.mchobby.be/index.php?title=MicroPython-Accueil)
- Dedicated page. A given project can have a page isolated from the remain of the Wiki as demonstrate the Cansat project page --> [https://wiki.mchobby.be/index.php?title=ENG-CANSAT-PICO-BELGIUM](https://wiki.mchobby.be/index.php?title=ENG-CANSAT-PICO-BELGIUM)
- A tutorial page (behind the "Boxed" entry point) can also have a navigator (see on the green box on the right side) allowing to jump quickly between the "chapters" --> [https://wiki.mchobby.be/index.php?title=ENG-CANSAT-PICO-THONNY](https://wiki.mchobby.be/index.php?title=ENG-CANSAT-PICO-THONNY)
- Finally I did wrote some Python code to extract navigator data and compile a PDF file from the Wiki -
 - see -> [https://wiki.mchobby.be/index.php?title=Pololu-Zumo-Shield-Arduino](https://wiki.mchobby.be/index.php?title=Pololu-Zumo-Shield-Arduino)
 - see the [compiled pdf file](https://df.mchobby.be/wiki-export/Pololu-Zumo-Shield-Arduino/Pololu-Zumo-Shield-Arduino.pdf).

I can offer:
- hosting for the documentation (into a separate page).
- Account creation for the Z80 Retro volunteers (filling the content).
- Creation of the pages/chapter structure (as well a "boxed" welcome page & Navigator).
- Guidance & support for writing chapter content with existing Wiki template I made for my own documentation. My [templates are documented](https://wiki.mchobby.be/index.php?title=Utilisateur:Admin)

As any wiki page, the page source is always available ;-)

Regards,
Dominique

**Comment by mchobby** [comment]
Aside my previous message, you should see how the [Commander-X16 did organized their documentation on Github with MarkDown files](https://github.com/X16Community/x16-docs/tree/master).
Project leader and participant have made a great job on writing the doc, the doc source is available and can be easily downloaded. 
A very nice approach!

**Comment by linuxplayground** [comment]
I started that in the manual repo. See: https://github.com/Z80-Retro/Z80-Retro-Manual

Feel free to raise pull requests. 

It's just the markdown at this point. So would need to add some way to compile the MD files into a pdf

**Comment by johnwinans** [comment]
Ya know..  @joelburton 's  point from almost two years ago has struck a chord with me this morning. 

As YouTube tries to add chapters to videos and we try to collect the current state of the project into a "manual" we don't have a catalog of what is where in the video series.  (It does not help that I don't script these things.  Someone once said something that rings in my head every time I post a video along the lines of "If I Had More Time, I Would Have Written a Shorter Letter.")

If anyone wants to participate in building an index of the topics and where they came from in the video series, I would LOVE to accept some pull requests!  These could go into a set of pages in the manual repo somewhere with links into the videos.

@Stefanie80 and everyone else I am sure is very correct in that the discord is a nightmare for finding the gems that are posted there by various folks amidst all the discussion... that, arguably has to happen in order to produce the gems.  I pin some of the things that I have run into.  But they would be much better if they were transcribed into something connected to the Retro project manual or something like that.

Anybody??

**Comment by johnwinans** [comment]
This has migrated into the [Z80-Retro-Manual](https://github.com/Z80-Retro/Z80-Retro-Manual) that @linuxplayground mentioned above.
